docroot/modules/contrib/custom_solr_search/custom_solr_search.module

```diff
@@ -31,17 +31,11 @@ function custom_solr_search_theme($existing, $type, $theme, $path) {
   return array (
     'custom_solr_search_result' => array(
       'variables' => array(
-        'url' => NULL,
         'title' => NULL,
         'author' => NULL,
         'publishDate' => NULL,
         'publisher' => NULL,
         'topic' => NULL,
-        'docid' => NULL,
```

Comment:
Don't we need docid here now? I believe this is the attribute being used in theme preprocessor to get node urls in the result listing.